core/src/main/scala/com/salesforce/op/stages/impl/feature/Transmogrifier.scala

```diff
@@ -608,8 +621,8 @@ trait MapPivotParams extends Params {
   protected def filterKeys[V](m: Map[String, V], shouldCleanKey: Boolean, shouldCleanValue: Boolean): Map[String, V] = {
     val map = cleanMap[V](m, shouldCleanKey, shouldCleanValue)
     val (whiteList, blackList) = (
-      $(whiteListKeys).map(cleanTextFn(_, shouldCleanKey)),
-      $(blackListKeys).map(cleanTextFn(_, shouldCleanKey))
+      $(whiteListKeys).map(cleanTextFn(_, shouldCleanKey, TransmogrifierDefaults.CleanParams)),
```

Comment:
and same here - replace ` TransmogrifierDefaults.CleanParams` with `shouldCleanValue`